LgiHdmiCec/LgiHdmiCec.cpp

```diff
@@ -937,7 +937,11 @@ namespace WPEFramework
             Frame.getBuffer(&input_frameBuf,&length);
 
             std::vector <char> buf;
-            buf.resize(length * 2);
+            // base64 encoded string uses 4 characters for every 3 bytes (using padding if necessary) - so assume padding is used
+            // Base64Encode doesn't seem to use base64 padding right now, but better to be future-proof
+            size_t required_buffer_size = 4 * (length / 3);
+            if (length % 3 != 0) required_buffer_size += 4;
```

Comment:
I can create follow-up task & refactor that after this PR is merged; I'd for now prefer to merge the code as-is, since this is the version that has been independently tested - is it ok with you?